interface_frontend.py

```diff
@@ -70,13 +71,34 @@ def init_window():
     x = x + i * 144
     cards_hand_coordinates.append((x, y))
 
+# dict card object distanc/rotation and fronend card name
+card_types_dict = {
+-1: 'back_up',
+1: 'move1',
+2: 'move2',
+3: 'move3',
+Rotation.U_TURN: 'u_turn',
+Rotation.LEFT: 'left',
+Rotation.RIGHT: 'right',
+}
```

Comment:
Tohle mě dost mate. V interface máš slovníky `cards_moves`, `cards_rotation`, které mají jako klíče právě hodnoty tohoto slovníku... Nebylo by lepší si ty karty při vytváření prostě pojmenovat? Tím myslím přidat další atribut třídy? Je to jenom nápad k zamyšlení :)